tools/generate_schema_wrapper.py

```diff
@@ -905,21 +907,31 @@ def _gen_artifacts(self) -> None:
                 tp_inner = f"OneOrSeq[{tp_inner}]"
             signature_args.append(f"{channel}: Optional[{tp_inner}] = Undefined")
             typed_dict_args.append(f"{channel}: {tp_inner}")
-            docstring_parameters.extend(
+            signature_docstring_parameters.extend(
                 (
                     f"{channel} : {', '.join(chain(docstring_union_types, it_rst_names))}",
                     f"    {process_description(info.deep_description)}",
                 )
             )
-        docstring_parameters += [""]
-        doc = indent_docstring(
-            docstring_parameters, indent_level=8, width=100, lstrip=False
+            typed_dict_docstring_parameters.extend(
+                (
+                    f"{channel} :",
+                    f"    {process_description(info.deep_description)}",
+                )
+            )
+        signature_docstring_parameters += [""]
+        typed_dict_docstring_parameters += [""]
+        signature_doc = indent_docstring(
+            signature_docstring_parameters, indent_level=8, width=100, lstrip=False
+        )
+        typed_dict_doc = indent_docstring(
+            typed_dict_docstring_parameters, indent_level=8, width=100, lstrip=False
```

Comment:
So I actually prefer the look of the shorter wrapping in `_EncodeKwds`, but I think you want the `indent_level=4`, since it is on the class and not the method

I thought I made the change to `8` in https://github.com/vega/altair/pull/3506 but I can't find it

Reply:
8 instead of 4 gives the additional indent marked in red:
<img width="257" alt="image" src="https://github.com/user-attachments/assets/a35066ec-2a02-4946-bdff-662a64572a6c">

Same as the `function_with_pep484_type_annotations ` example in https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html which also does not have type hints in the docstring. Or maybe I'm misreading your message?

Reply:
No that part is good - its the right hand side wraps early, thinking its hit 100

Reply:
> So I actually prefer the look of the shorter wrapping in `_EncodeKwds`, but I think you want the `indent_level=4`, since it is on the class and not the method @binste
> 
> I thought I made the change to `8` in #3506 but I can't find it

So the bottom one should be able to extend further.
But I was saying I prefer wrapping earlier, although it doesn't match the style elsewhere in the docs.

I think 100 is a jupyter thing

![image](https://github.com/user-attachments/assets/bfa11a08-258d-4212-b0cc-e57a4c4b2a05)

Reply:
This had less of an impact than I expected but we have 4 extra chars of width in [`bac1f67` (#3515)](https://github.com/vega/altair/pull/3515/commits/bac1f67c97760c00d13d2b3100a468c484a36946)